docs/source/routing/self-hosted/containerization/aws.mdx

```diff
@@ -0,0 +1,162 @@
+    ```
+
+    - Because Cloud Run only supports AMD64-based images, the `docker buildx build --platform linux/amd64` command ensures the image is built for AMD64 and is compatible.  
+    - The `--load` option loads the built image to `docker images`.
+
+1. Run `docker images` and validate that your router image is in the returned list of images.
+
+## Push router image to registry
+
+Now that you have a built router image, create a repository in Elastic Cloud Registry (ECR) and push your image to it:
+
+1. In a local terminal, run the AWS CLI command to create a new ECR repository:
+    - For `--repository-name`, set a name for your repository (for example, `router-repo`) 
+    - For `--region`, set your AWS region (for example, `us-west-1`)
+
+    ```bash showLineNumbers=false
+    aws ecr create-repository \
+        --repository-name router-repo \
+        --region us-west-1
+    ```
+
+1. In AWS CLI, authenticate your Docker CLI to ECR.
+    - For `--region`, use your AWS regions (for example, `us-west-1`)
+    - Use your ECR repository URI, which you can copy from your ECR Repositories Console (for example, `0123456789000.dkr.ecr.us-west-1.amazonaws.com`)
+
+    ```bash showLineNumbers=false
+    aws ecr get-login-password --region us-west-1 | docker login --username AWS --password-stdin 0123456789000.dkr.ecr.us-west-1.amazonaws.com
+    ```
+
+    > To troubleshoot ECR authentication, go to [AWS documentation](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-cli.html#cli-authenticate-registry).  
+
+1. Run `docker tag` to tag the image before pushing it to ECR.
+
+    ```bash showLineNumbers=false
+    docker tag router-aws:v1.61.0 0123456789000.dkr.ecr.us-west-1.amazonaws.com/router-repo:v1.61.0
+    ```
+
+1. Run `docker push` to push the router image to your ECR repository URI, using a tag (e.g., `:v1.61.0`):
+
+    ```bash showLineNumbers=false
+    docker push 0123456789000.dkr.ecr.us-west-1.amazonaws.com/router-repo:v1.61.0
+    ```
+
+1. Run `aws ecr list-images` and validate that your image is in the list of images in your ECR repository:
+
+    ```bash showLineNumbers=false
+    aws ecr list-images --repository-name router-repo
+    ```
+
+## Create and deploy ECS task
+
+With your image pushed to your ECR repository, in ECS you can define a task for the router and deploy it as a service.
+
+### Create cluster
+
+You need an ECS cluster to deploy the router.
+
+If you don't have a cluster, you can create one with default settings: 
+
+1. In the AWS Console, go to the Amazon ECS Console, then click **Create cluster**.
+1. Enter a name for your cluster.
+1. Click **Create**.
+
+### Create task definition
+
+Create an ECS task definition for your router:
+
+1. In the AWS ECS Console, go to **Task definitions** from the left navigation panel, then click **Create new task definition** and select **Create new task definition**.
+1. Fill in the details for **Container - 1**:
```

Comment:
Consider using a table to more clearly show the necessary input values. (E.g. the `rover init` values in https://www.apollographql.com/docs/guides/rest/setup#create-a-graph)